db/table_cache.cc

```diff
@@ -98,7 +98,11 @@ Status TableCache::GetTableReader(
 
   RecordTick(ioptions_.statistics, NO_FILE_OPENS);
   if (s.ok()) {
-    if (readahead > 0) {
+    if (readahead > 0 && !env_options.use_mmap_reads) {
+      // Not useful for (nor compatible with) mmap files since
```

Comment:
I'm not totally sure it's not useful. I think we don't need to draw this conclusion it here.

Reply:
Thanks I will change it to "not compatible with".